src/unittest/python/plugins/python/cram_plugin_tests.py

```diff
@@ -17,6 +17,20 @@
 #   limitations under the License.
 
 import unittest
+import sys
+
+try:
+    from unittest import skipIf
+except ImportError:
+    def skipIf(*args, **kwargs):
+        def inner(test):
```

Comment:
Can we use `functools.wraps` for this? https://docs.python.org/2/library/functools.html